src/styles/blocks/stars.scss

```
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

u can use just [background](https://developer.mozilla.org/en-US/docs/Web/CSS/background)
```suggestion
    background: ............;
```